### PR TITLE
Minor/trivial fix for consistency

### PR DESCRIPTION
**Pull request by illgitthat**
Change on-premise and on premises to on-premises for consistency.

Reference: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/cloud-computing-terms

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbHybrid/hybrid/move-users-from-the-cloud-to-on-premises.md [hunk 1]
```diff
@@ -15,40 +15,40 @@ ms.collection:
...
 - Skype for Business hybrid must be configured, as described in [Configure Skype for Business hybrid](configure-federation-with-skype-for-business-online.md).
 
-## Moving users back to on premises
+## Moving users back to on-premises
 
 Once you move a user from the cloud back to on-premises:
 
 - The user interacts with your Skype for Business Server deployment for its functionality. 
-- Any contacts that existed in either Skype for Business Online or Teams are migrated  to Skype for Business Server. The two sets of contacts are merged and then migrated back to on premises.  In addition, contacts that are pre-existing in Teams remain in Teams.
+- Any contacts that existed in either Skype for Business Online or Teams are migrated  to Skype for Business Server. The two sets of contacts are merged and then migrated back to on-premises.  In addition, contacts that are pre-existing in Teams remain in Teams.
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/move-users-from-the-cloud-to-on-premises.md]:
```suggestion
- Any contacts that existed in either Skype for Business Online or Teams are migrated  to Skype for Business Server. The two sets of contacts are merged and then migrated back to on-premises. In addition, contacts that are pre-existing in Teams remain in Teams.
```

File: Skype/SfbHybrid/hybrid/move-users-from-the-cloud-to-on-premises.md [hunk 2]
```diff
@@ -15,40 +15,40 @@ ms.collection:
...
 - If the user also uses Teams, they will not have the ability to interoperate with Skype for Business users, nor will they be able to communicate with users in federated organizations.
-- Meetings in Skype for Business Online are *not* automatically migrated back to on premises. Users should either reschedule their meetings or, if desired, use the [Meeting Migration Tool](https://support.office.com/en-us/article/2b525fe6-ed0f-4331-b533-c31546fcf4d4).
+- Meetings in Skype for Business Online are *not* automatically migrated back to on-premises. Users should either reschedule their meetings or, if desired, use the [Meeting Migration Tool](https://support.office.com/en-us/article/2b525fe6-ed0f-4331-b533-c31546fcf4d4).
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/move-users-from-the-cloud-to-on-premises.md]:
```suggestion
- Meetings in Skype for Business Online are *not* automatically migrated back to on-premises. Users should either reschedule their meetings or, if desired, use the [Meeting Migration Tool](https://support.office.com/article/2b525fe6-ed0f-4331-b533-c31546fcf4d4).
```

**Comment by scanum** [comment]
@CarolynRowe Hi Carolyn! 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**